ShippingApp/src/main/java/com/javaschool/shipping/ShippingAppApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.javaschool.shipping;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication(scanBasePackages = {"com.javaschool"})
```

Comment:
Is it necessary `scanBasePackages` on this class?

Reply:
For the structure of the project, yes. With the @SpringBootApplication only sub-packages of where the class is scanned, an alternative would be to move the class up one level.